codepipeline-tf/terraform.auto.tfvars

```diff
@@ -0,0 +1,44 @@
+##################################################
+# tfvars
+##################################################
+
+
+namespace_name = "example"
+account_id     = "your-account-id"
+domain_name    = "your-domain"
+
+s3_bucket_name = "terraform-state-xc3-example-pipeline"
+
+
+xc3_codepipeline_role  = "arn:aws:iam::201635854701:role/xccc-pipeline-role"
```

Comment:
this should be dummy and value shoud be imported via sh script that runs on 1st time

Reply:
this was for testing purpose, now we've tested roles with terraform resources and almost testing is completed. Will pass the arn of dynamically build roles on runtime.